crates/runtime/src/http/v1/nsql.rs

```diff
@@ -190,6 +193,7 @@ LIMIT 5
         )))
     )
 ))]
+#[allow(clippy::too_many_lines)]
```

Comment:
Did attempt to refactor but was unable to come up with good approach to decouple/simplify method logic. Most logic is converting errors to response message.